vrk-kpa/suomidigi
# SUOPA-831 Profile page

To test: 
- `make fresh`
- Log in and go to https://suomidigi.docker.sh/ryhmat/digituki 
   - On the page there's a small link on the right side of the page called "Liity ryhmään". Click on it and then click the confirmation button to join the group. 
- When you've joined the group, go to your own page https://suomidigi.docker.sh/user/ and check that there is a list of groups you've joined.
